aws_signing_helper/credentials.go

```diff
@@ -50,31 +54,34 @@ func GenerateCredentials(opts *CredentialsOpts) (CredentialProcessOutput, error)
 	if opts.Region == "" {
 		opts.Region = trustAnchorArn.Region
 	}
-
-	privateKey, err := ReadPrivateKeyData(opts.PrivateKeyId)
-	if err != nil {
-		return CredentialProcessOutput{}, err
-	}
-	certificateData, err := ReadCertificateData(opts.CertificateId)
-	if err != nil {
-		return CredentialProcessOutput{}, err
-	}
-	certificateDerData, err := base64.StdEncoding.DecodeString(certificateData.CertificateData)
-	if err != nil {
-		return CredentialProcessOutput{}, err
-	}
-	certificate, err := x509.ParseCertificate([]byte(certificateDerData))
-	if err != nil {
-		return CredentialProcessOutput{}, err
+	var LibPkcs11 = opts.LibPkcs11
```

Comment:
Somewhat minor, but I think it might be better to use the fields within the `opts` structure directly, as opposed to assigning them to new variables before using them.

Reply:
Removed the new variable to use the opts structure directly.